### DISCUSSION TITLE
Near-Wall Grid Resolution - y+ (yplus)

### DISCUSSION THREAD

**Discussion by UnknownFdsUser** [firemodels/fds#13517] [Q&A]
Good morning everyone, 

we know that turbulent flows are a ubiquitous phenomenon in CFD and are significantly affected by the presence of walls, where regions affected by viscosity have large gradients in solution variables. If I understand correctly, an accurate representation of the near-wall region determines a successful prediction of the turbulent flows bounded by the wall. This representation is given by the parameter y+, which in FDS is covered on the Verification Guide section 5.4 (page 61). y+ can be monitored in FDS by the command 
&BNDF QUANTITY='VISCOUS WALL UNITS'/.

Also, y+ is covered in the User Guide on p. 387, if I understand correctly the lower the y+ parameter the higher the degree of resolution for turbulent flows. For engineering analysis and LES turbulence model, a value of y+=100 is acceptable. 

This value must be monitored at each obstruction, right? 

If I model a building, every obstruction that creates flows should have a y+=100, not just the walls, right?

**Comment by rmcdermo** [comment]
The wall function used by FDS comes from the law of the wall and thus assumes a steady, well-developed turbulent flow without buoyancy or other complications.  Monitoring y+ does not hurt, but it also does not guarantee accuracy.  Like many LES quality metrics, y+ should not be the only thing you look at, it is a rough guide.  You need to think about the controlling length scales in your simulation and make sure these are resolved, usually with 10 or more cells.  Grid resolution studies and comparison with validation data are usually more helpful than dogmatically relying on LES quality metrics.

In a fire simulation, the most important output is the heat release rate (see CHID_hrr.csv file).  This is controlled by the mass flux of fuel and the ventilation.  Ventilation often depends on door and window openings and/or HVAC ventilation.  The y+ for these components of the simulation are usually irrelevant.  In other words, don't get too hung up on y+ unless you are really trying to resolve a boundary layer flow.

If you have a large building, outputting BNDF files can get expensive, so if you do this, I'd set DT_BNDF to something large enough not to slow you down.

**Reply by UnknownFdsUser** [reply]
Thank you for your response Dr. McDermott! All clear.
